src/DurableTask.Netherite/Events/README.md

```diff
@@ -0,0 +1,53 @@
+# Netherite Events
+
+This directory contains the event definitions for the Netherite engine. All components in Netherite communicate via events over an event bus (i.e. Azure Event Hubs). The events are defined as .NET classes that inherit from the `DurableTask.Netherite.Event` abstract base class.
+
+Event classes are organized on the file system based on where they execute. For example `ClientEvents` are events that execute on the client and `LoadMonitor` events are executed on the global load monitor component.
+
+## Event types
+
+The following are the different event types supported by Netherite.
+
+### [Client events](./ClientEvents/)
+
+Client events are events that are sent from the partition and *executed on the client*. In most cases, they are responses to client-initiated requests. For example, a client sends a [StateRequestReceived](./PartitionEvents/External/FromClients/StateRequestReceived.cs) event to a partition to request the state of an orchestration, and the partition responds back to the client with a [StateResponseReceived](./ClientEvents/StateResponseReceived.cs) event, containing that result.
```

Comment:
Something confuses me here: This says that client events are **sent from the partition** and **executed on the client**, but then you explain that `StateRequestReceived` is **sent from the partition** and **executed in the partition** which seems contradictory.

Perhaps the way to make sense of this is that `StateRequestReceived` is not considered a client event? If so - I recommend we call that out for clarity's sake :-) .

Reply:
Yeah, I was worried that readers might be confused by this, and you confirmed it. :) Let me try to clarify this...